FakeItEasy.sln

```diff
@@ -1,26 +1,32 @@
-﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio 2013
-VisualStudioVersion = 12.0.31101.0

```

Comment:
> If we'd like to consciously make 2015 the default, I'm happy to discuss.

It occurs to me that we've discussed. I'd be content with an update to the description of the PR to indicate that that change came along with the others, if we want to just move ahead.